html/js/tree/Move.js

```diff
@@ -0,0 +1,34 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Move.
+    * @class
+    * @param {Node} parent Parent node origin of the move.
+    * @param {Node} child Child node destiny of the move.
+    */
+    function Move (parent, child) {
+        this.parent = parent;
+        parent.children[child] = this;
```

Comment:
I don't understand this line. Maybe it was part of a previous and discarded  alternative implementation of Moves?